convene-web/app/lib/utilities.rb

```diff
@@ -0,0 +1,25 @@
+# Utilities are External Services that provide functionality to a piece of
+# Furniture.
+#
+# They are connected to a {Space} via a {UtilityConnection}, which securely
+# stores any API keys or similar configuration data.
+#
+# Examples of a Utility may be:
+# - A Jitsi-Meet or BigBlueButton instance for videoconferencing
+# - An Airtable Base or Google Apps account that acts as a human-editable
+#   backend-data store for a piece of Furniture.
+# - A Dropbox, OneCloud or Google Drive account for file sharing.
+# - ???
+#
+module Utilities
+  # @return [Utility] configured as appropriate given the connection
+  def self.from_connection(connection)
+    Utility.new(settings: connection)
```

Comment:
Sounds like the "connection" is actually a "configuration" or "settings" (as opposed to a server-to-server connection, which is what I would have assumed from the name).

Reply:
Yea, that's a good point